Mathlib/Topology/Algebra/Order/Compact.lean

```diff
@@ -263,6 +263,17 @@ theorem IsCompact.exists_isMinOn [ClosedIicTopology α] {s : Set β} (hs : IsCom
   rcases (hs.image_of_continuousOn hf).exists_isLeast (ne_s.image f) with ⟨_, ⟨x, hxs, rfl⟩, hx⟩
   exact ⟨x, hxs, ball_image_iff.1 hx⟩
 
+/-- The **extreme value theorem**: a continuous function realizes its minimum on a compact set `s`.
+  This variant is for functions into a `NoMaxOrder`, without the assumption `s.Nonempty`. -/
```

Comment:
Please rewrite the docstring. This lemma states nothing about realizing its minimum. Maybe "if a continuous function lies strictly above `a` on a compact set, then it has a lower bound strictly above `a`."

bors d+

Reply:
Good point and a much better docstring; thank you.